prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -307,23 +316,16 @@ struct fi_ibv_rdm_tagged_postponed_entry {
 	struct fi_ibv_rdm_tagged_conn *conn;
 };
 
-enum fi_ibv_rdm_buffer_status {
-	BUF_STATUS_FREE = 0,
-	BUF_STATUS_BUSY
-};
-
+/*
+ * Size of the structure is performance sensitive. TODO: resize to 4 bytes.
+ */
 struct fi_ibv_rdm_buffer_service_data {
 	volatile enum fi_ibv_rdm_buffer_status status;
-	int seq_number;
+	int pkt_len;
 };
 
-#define FI_IBV_RDM_BUFF_SERVICE_DATA_SIZE				\
-	(sizeof(struct fi_ibv_rdm_buffer_service_data)			\
-		< FI_IBV_RDM_MEM_ALIGNMENT ? FI_IBV_RDM_MEM_ALIGNMENT	\
-		: (sizeof(struct fi_ibv_rdm_buffer_service_data) +	\
-		  (FI_IBV_RDM_MEM_ALIGNMENT -				\
-		   sizeof(struct fi_ibv_rdm_buffer_service_data) %	\
-		   FI_IBV_RDM_MEM_ALIGNMENT)))
+#define FI_IBV_RDM_BUFF_SERVICE_DATA_SIZE	\
+	(sizeof(struct fi_ibv_rdm_buffer_service_data))
```

Comment:
Having a define replace sizeof unnecessarily complicates the code. Just use sizeof directly.